scripts/plot_v_score_rel_err.py

```diff
@@ -49,13 +49,7 @@ def main():
 
         method = row[2]
         # Skip some J2 values to make the plot not too crowded
-        if ham_type == "TFIsing":
-            if "VQE" in method and method not in [
-                "VQE HV (d = 26)",
-                "VQE R-CX (d = 10)",
-            ]:
-                continue
-        elif ham_type == "J1J2":
+        if ham_type == "J1J2":
```

Comment:
As Guglielmo cleaned up his results in `TFIsing/chain_10_O_1.md`, we don't need to manually filter them